lib/constants.js

```diff
@@ -56,6 +56,7 @@ module.exports = {
 
     AMQP_DEFAULT_EXCHANGE: 'amq.topic',
     AMQP_DEFAULT_QUEUE: 'iotaqueue',
+    AMQP_DEFAULT_DURABLE: true,
```

Comment:
I understand that the config.amqp.durable (= AMQP_DEFAULT_DURABLE env var) was already included in config.js and documentation but up to now the code didn't use it. Is my understanding correct?

Reply:
Yes, that is

Reply:
NTC